gitbook-docs/security.md

```diff
@@ -31,11 +42,11 @@ The client may send the login request with a `Content-Type` of `application/json
 ```
 
 In response to a valid login, the server shall respond with a 200 (OK) status, set an HTTP session cookie and include
-the token type and the token value in the body of the response. The response `Content-Type` must be `application/json`.
+the token expiry in seconds and the token value in the body of the response. The response `Content-Type` must be `application/json`.
 
 ```json
 {
-  "type": "JWT",
+  "expiry": 86400,
```

Comment:
Should be timeToLive.

Reply:
It is in commit 6047ff6 above

Reply:
That changes line 80, this is line 49...